repositories/MerchStoreRepository.ts

```diff
@@ -53,11 +54,15 @@ export class MerchItemRepository extends BaseRepository<MerchandiseItemModel> {
     return this.repository.save(item);
   }
 
-  public async updateMerchItemInCollection(collection: string, hidden: boolean): Promise<void> {
+  public async updateMerchItemsInCollection(collection: string, changes: any): Promise<void> {
+    if (Object.keys(changes).length === 0) {
+      throw new UserError('No changes sent');
+    }
```

Comment:
I appreciate that you're being defensive here but we're the ones calling this method so if no changes are sent, then that's entirely on us, so returning a UserError (4xx) is a bit disingenuous. TBH I think it's fine to remove this check since every time we call it we're manually setting the keys in `changes`. If we ever get to the point where we're passing in changes from the request (e.g. user sends a bunch of fields to change for all the merch items in the collection), then I think we can add this back.

Reply:
Got it, should be removed